lib/scrimp/app.rb

```diff
@@ -88,7 +101,8 @@ class App < Sinatra::Base
       end.compact
 
       begin
-        transport = Thrift::FramedTransport.new Thrift::Socket.new(invocation['host'], invocation['port'])
+        socket = Object::const_get(invocation['socket']).new(invocation['host'], invocation['port'])
+        transport = Object::const_get(invocation['transport']).new(socket)
```

Comment:
We should provide defaults for the new fields to keep this passive, i.e. `invocation['transport'] || 'Thrift::FramedTransport'`, because there are some users who post request json that they've saved previously.